modules/components/pages/processors/cohere_chat.adoc

```diff
@@ -0,0 +1,487 @@
+= cohere_chat
+// tag::single-source[]
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Generates responses to messages in a chat conversation, using the Cohere API.
+
+ifndef::env-cloud[]
+Introduced in version 4.37.0.
+endif::[]
+
+[tabs]
+======
+Common::
++
+--
+```yml
+# Common config fields, showing default values
+label: ""
+cohere_chat:
+  base_url: https://api.cohere.com
+  auth_token: "" # No default (required)
+  model: command-r-plus # No default (required)
+  prompt: "" # No default (optional)
+  system_prompt: "" # No default (optional)
+  max_tokens: 0 # No default (optional)
+  temperature: 0 # No default (optional)
+  response_format: text
+  json_schema: "" # No default (optional)
+```
+--
+Advanced::
++
+--
+```yml
+# All config fields, showing default values
+label: ""
+cohere_chat:
+  base_url: https://api.cohere.com
+  auth_token: "" # No default (required)
+  model: command-r-plus # No default (required)
+  prompt: "" # No default (optional)
+  system_prompt: "" # No default (optional)
+  max_tokens: 0 # No default (optional)
+  temperature: 0 # No default (optional)
+  response_format: text
+  json_schema: "" # No default (optional)
+  schema_registry:
+    url: "" # No default (required)
+    subject: "" # No default (required)
+    refresh_interval: "" # No default (optional)
+    tls:
+      skip_cert_verify: false
+      enable_renegotiation: false
+      root_cas: ""
+      root_cas_file: ""
+      client_certs: []
+    oauth:
+      enabled: false
+      consumer_key: ""
+      consumer_secret: ""
+      access_token: ""
+      access_token_secret: ""
+    basic_auth:
+      enabled: false
+      username: ""
+      password: ""
+    jwt:
+      enabled: false
+      private_key_file: ""
+      signing_method: ""
+      claims: {}
+      headers: {}
+  top_p: 0 # No default (optional)
+  frequency_penalty: 0 # No default (optional)
+  presence_penalty: 0 # No default (optional)
+  seed: 0 # No default (optional)
+  stop: [] # No default (optional)
+```
+--
+======
+
+This processor sends the contents of user prompts to the Cohere API, which generates responses. By default, the processor submits the entire payload of each message as a string, unless you use the `prompt` field to customize it.
```

Comment:
> By default, the processor submits the entire payload of each message as a string, unless you use the `prompt` field to customize it

there's no example of how to do this on the page.

Reply:
@rockwotj. Could you provide an example for how to select/specify the text strings to submit? The `cohere_embeddings` and `gcp_vertex_ai_embeddings` also seem to work in the same way.

Reply:
Added to resolve this: https://redpandadata.atlassian.net/browse/DOC-644